README.md

```diff
@@ -83,6 +83,9 @@ API key support will be added in a future update.  With an API key, the usage li
 - Unlimited/month
 - $.0001/request over the free usage limit, rounded up to the nearest cent
 
+## Note
+
+The code in this repository is built for Deephaven Community Core v0.13.0. No guarantee of forwards or backwards compatibility is given.
```

Comment:
Add a blank line between lines 88 and 89. Also, since this pulls from an external API, the note should mention that the code that pulls from it may be outdated as well.